spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/SparkHiveTable.java

```diff
@@ -55,4 +59,10 @@ public Map<String, String> properties() {
   public Transform[] partitioning() {
     return gravitinoTableInfoHelper.partitioning();
   }
+
+  /** to keep consistent behavior with SparkIcebergTable. */
```

Comment:
I think there's no need to keep consistent with Iceberg

Reply:
updated